src/components/fidelity_bond/fb_utils.ts

```diff
@@ -0,0 +1,79 @@
+import { Lockdate } from '../../libs/JmWalletApi'
+
+type Milliseconds = number
+
+export type YearsRange = {
+  min: number
+  max: number
+}
+
+export const toYearsRange = (min: number, max: number): YearsRange => {
+  if (max <= min) {
+    throw new Error('Invalid values for range of years.')
+  }
+  return { min, max }
+}
+
+// A maximum of years for a timelock to be accepted.
+// This is useful in simple mode - when it should be prevented that users
+// lock up their coins for an awful amount of time by accident.
+// In "advanced" mode, this can be dropped or increased substantially.
+export const DEFAULT_MAX_TIMELOCK_YEARS = 10
```

Comment:
10 years is quite a lot 😅  -- that being said, I don't know what a sane maximum should be. 10 years is probably good!

Reply:
Yeah.. as far as I can see most fidelity bonds are not beyond the year 2026.. should we decrease to a maximum of 5 years and wait for user feedback?